sdks/python/apache_beam/runners/dataflow/internal/apiclient.py

```diff
@@ -436,6 +436,19 @@ def create_job_description(self, job):
     # TODO(silviuc): Remove the debug logging eventually.
     logging.info('JOB: %s', job)
 
+  @retry.with_exponential_backoff()  # Using retry defaults from utils/retry.py
```

Comment:
1. Do we need the `# Using retry defaults from utils/retry.py` comment?
2. Is the default good for this case?